<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Which section do you want to manage?</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>33</y>
       <width>113</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="employeeRB">
        <property name="text">
         <string>Emolyees</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="DepartmentRB">
        <property name="text">
         <string>Departments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ClientsRB">
        <property name="text">
         <string>Clients</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ProductsRB">
        <property name="text">
         <string>Products</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Buys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ManufactureRB">
        <property name="text">
         <string>Manufacture</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Welcome To MOMAM Pharmaceutical Company</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>371</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Connecting to Database</string>
    </property>
    <widget class="QPushButton" name="connect">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>34</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="username">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>       User Name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Password">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>        Password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
